src/refmvs.rs

```diff
@@ -32,44 +32,44 @@ use zerocopy::FromZeroes;
 
 #[derive(Clone, Copy, Default, PartialEq, Eq)]
 #[repr(C, packed)]
-pub struct refmvs_temporal_block {
-    pub mv: mv,
+pub struct RefMvsTemporalBlock {
+    pub mv: Mv,
     pub r#ref: i8,
 }
 
 #[derive(Clone, Copy, PartialEq, Eq, FromZeroes)]
 #[repr(C)]
-pub struct refmvs_refpair {
+pub struct RefMvsRefPair {
     pub r#ref: [i8; 2],
 }
 
-impl From<[i8; 2]> for refmvs_refpair {
+impl From<[i8; 2]> for RefMvsRefPair {
     fn from(from: [i8; 2]) -> Self {
-        refmvs_refpair { r#ref: from }
+        RefMvsRefPair { r#ref: from }
     }
 }
 
 #[derive(Clone, Copy, Default, PartialEq, Eq, FromZeroes)]
 #[repr(C)]
-pub struct refmvs_mvpair {
-    pub mv: [mv; 2],
+pub struct RefMvsMvPair {
+    pub mv: [Mv; 2],
 }
 
 #[derive(Clone, Copy, FromZeroes)]
 #[repr(C, align(4))]
-pub struct refmvs_block {
-    pub mv: refmvs_mvpair,
-    pub r#ref: refmvs_refpair,
+pub struct RefMvsBlock {
+    pub mv: RefMvsMvPair,
+    pub r#ref: RefMvsRefPair,
     pub bs: BlockSize,
     pub mf: u8,
 }
 
 // In C, this is packed and is 12 bytes.
 // In Rust, being packed and aligned is tricky
-const _: () = assert!(mem::size_of::<refmvs_block>() == 12);
+const _: () = assert!(mem::size_of::<RefMvsBlock>() == 12);
 
 #[repr(C)]
-pub(crate) struct refmvs_frame<'a> {
+pub(crate) struct AsmRefMvsFrame<'a> {
```

Comment:
Did `RefMvsFrame` conflict with another type? This seems to be the only one with `Asm`?

Reply:
Yeah, we have a safe non-asm version of this called `RefMvsFrame` already, and this is the version to pass to asm.